.github/workflows/ci.yml

```diff
@@ -297,14 +297,14 @@ jobs:
         run: make run-all-unit-test
       - name: Crate version check
         if: matrix.check == 'crate-version'
-        run: ./release-scripts/test-crate-version.sh
+        run: ./release-tools/test-crate-version.sh
```

Comment:
You should run your tool to perform this check

Reply:
Implemented